tests/negative/renderpass.cpp

```diff
@@ -2336,6 +2336,53 @@ TEST_F(NegativeRenderPass, Framebuffer) {
     m_errorMonitor->VerifyFound();
 }
 
+TEST_F(NegativeRenderPass, FramebufferAttachmentPointers) {
+    TEST_DESCRIPTION("pAttachments points to valid objects for the Framebuffer creation");
+
+    ASSERT_NO_FATAL_FAILURE(Init());
+    ASSERT_NO_FATAL_FAILURE(InitRenderTarget(2));
+
+    // Create a renderPass with a single color attachment
+    VkAttachmentReference attachment_ref = {};
+    attachment_ref.layout = VK_IMAGE_LAYOUT_GENERAL;
+    std::array<VkAttachmentReference, 2> attachment_refs = {attachment_ref, attachment_ref};
```

Comment:
Nit: could simply be:
```suggestion
    std::array attachment_refs = {attachment_ref, attachment_ref};
```

Reply:
agree, that is nicer